packages/PowerGrid/src/cell-types/bar-cell-renderer.ts

```diff
@@ -9,18 +9,23 @@ interface BarCellSettings {
 const BarCellSettingsProperties = {
   color: DG.Property.js('color', TYPE.STRING),
   radius: DG.Property.js('radius', TYPE.STRING),
-}
+};
 
 export class BarCellRenderer extends DG.GridCellRenderer {
-  get name() { return 'bar'; }
-  get cellType() { return 'bar'; }
+  get name() {return 'bar';}
+
+  get cellType() {return 'bar';}
```

Comment:
Should be surrounded with spaces, like previously: `get cellType() { return 'bar'; }`